spyder_notebook/widgets/client.py

```diff
@@ -58,11 +58,13 @@ def contextMenuEvent(self, event):
         """Don't show some actions which have no meaning for the IPython
         notebook."""
         menu = QMenu(self)
-        actions = [self.pageAction(QWebEnginePage.SelectAll),
-                   self.pageAction(QWebEnginePage.Copy), None,
-                   self.zoom_in_action, self.zoom_out_action]
+        plugin_actions = self.parent().plugin_actions
+        actions = plugin_actions + [None,
+                                    self.pageAction(QWebEnginePage.SelectAll),
+                                    self.pageAction(QWebEnginePage.Copy), None,
+                                    self.zoom_in_action, self.zoom_out_action]
         if not WEBENGINE:
-            settings = self.page().settings()
```

Comment:
After making the changes I suggested to `DOMWidget`, you can restore this line to its previous state.